app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -47,6 +64,7 @@ public function store(Request $request)
     {
         $this->validate($request, [
             'internal_title' => 'required|string|unique:campaigns',
+            'cause' => 'required|string',
```

Comment:
For now, I only put validation to make sure the cause is a string and is required since all causes we'd want come from the drop down menu that is hardcoded (there is no way to enter your own cause and no API route to write this in manually either). Because of this, I didn't think there was need for validation for the cause to be within specific parameters since the drop down values has that covered. 

I'll make a note to myself though to check the values in Ashes to make sure there aren't any additional cause values besides what is in the array above so we have consistent data when we migrate.

Reply:
@chloealee Is there a way to make the `No cause` choice blank so that it wouldn't register as a string and we wouldn't have to worry about folks submitting `No cause`? Or maybe we actually don't need to worry about it so much since can change the cause later and only a small number of people will be creating campaigns in Rogue

Reply:
@katiecrane great call! Updated that here! https://github.com/DoSomething/rogue/pull/780#discussion_r233221919